src/config.rs

```diff
@@ -63,10 +81,37 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
     let ipc_channel = matches.opt_str("ipc-channel");
     let no_panel = matches.opt_present("no-panel");
 
+    let width = matches.opt_get::<u32>("width").unwrap_or_else(|e| {
+        log::error!("Failed to parse width command line argument: {e}");
```

Comment:
This can just be an info log. We will fallback to default size (800x600 perhaps)

Reply:
This is a parsing error, not a missing value (e.g. they passed in `--width=abc`)